### PR TITLE
[kueuectl] Added Node Labels column on resource flavor list.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
Added `Node Labels` column on resource flavor list on kueuectl.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
CLI: Added Node Labels column on resource flavor list.
```

### CODE REVIEW

File: site/static/examples/admin/single-clusterqueue-setup.yaml [hunk 1]
```diff
@@ -2,6 +2,11 @@ apiVersion: kueue.x-k8s.io/v1beta1
 kind: ResourceFlavor
 metadata:
   name: "default-flavor"
+spec:
+  nodeLabels:
+    key1: value
+    key2: value
+
```
Comment (alculquicondor) [inline site/static/examples/admin/single-clusterqueue-setup.yaml:9]: I just realized that this wasn't a good idea....
The example can no longer be used in a base cluster that doesn't have these labels.

Was this change necessary for the test?

Comment (mbobrovskyi) [reply site/static/examples/admin/single-clusterqueue-setup.yaml:9]: I think I missed to remove it after test. I can revert it.

Comment (mbobrovskyi) [reply site/static/examples/admin/single-clusterqueue-setup.yaml:9]: Fixed on #2928.
